osidb/models.py

```diff
@@ -1814,7 +1814,9 @@ class TrackerManager(ACLMixinManager, TrackingMixinManager):
     """tracker manager"""
 
     @staticmethod
-    def create_tracker(affect, external_system_id, _type, **extra_fields):
+    def create_tracker(
+        affect, external_system_id, _type, raise_validation_error=True, **extra_fields
```

Comment:
not a big fan of bubbling up `raise_validation_error`... but don't see an alternative other than rewriting the convertor to not use `create_tracker`.